### PR TITLE
Fix GN helper script PYTHONPATH assert misfiring

### PR DESCRIPTION
**Pull request by arjungm**
This helper script was checking that the directory it was invoked from was also on the path. However this check didn't account for windows path style issues. This change makes it robust to those differences.

b/299098707

### CODE REVIEW

**Comment by arjungm** [comment]
Note: without this fix, locally running docker containers with docker-compose is broken.

**Review by niranjanyardi — APPROVED** [review]
lgtm, might be good to see oscar's opinion too before submitting

**Comment by oxve** [comment]
This check might actually belong in GN. It's GN that needs the python files on the path and we will get the error even with this change if gn is invoked without gn.py, e.g. when running ninja with changes to gn files.

**Review by oxve — APPROVED** [review]

File: cobalt/build/gn.py [hunk 1]
```diff
@@ -51,10 +51,11 @@ def main(out_directory: str, platform: str, build_type: str,
 
 
 if __name__ == '__main__':
-  current_working_dir = os.getcwd()
-  python_env_var = os.getenv('PYTHONPATH')
+  current_working_dir = os.path.realpath(os.getcwd())
```
Comment (niranjanyardi) [inline cobalt/build/gn.py:54]: This is a good change, hoever given that oscar is moving the check to GN: https://github.com/youtube/cobalt/pull/1536

changing it here is obsolete, you dont need the check in 2 places (python + GN)

@oxve  should ideally use this condition in his script.

**Review by niranjanyardi — APPROVED** [review]

**Pull request closed without merging**